<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MusicPlayer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>qrc/dasturbek.png</normaloff>qrc/dasturbek.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QFrame" name="frame_3">
      <property name="minimumSize">
       <size>
        <width>800</width>
        <height>55</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>55</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_11">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>7</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="0" column="0">
        <widget class="QPushButton" name="play_video_4">
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">.QPushButton {
  background-color: rgb(255, 255, 255, 0);
  border: none;
  color: #1A6DFF;
  text-align: left;
  font-size: 28px;
  padding: 5px;
  margin:0 0 0 0;
}
</string>
         </property>
         <property name="text">
          <string>MusicPlayer</string>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <widget class="QFrame" name="frame_5">
         <property name="maximumSize">
          <size>
           <width>290</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgba(1, 57, 44, 0.1);
border: 1px  solid rgb(26, 109, 255, 0.6);
border-radius: 20px;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="ism">
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">height:40px;
color: #1A6DFF;
font-size:24px;
background-color: rgba(1, 57, 44, 0);
border:none;
padding-left: 5px</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="placeholderText">
             <string>Search</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="stop_music_2">
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">.QPushButton {
  background-color: rgb(255, 255, 255, 0);
  border: none;
  color: #1A6DFF;
  text-align: center;
  font-size: 22px;
  padding: 5px;
  margin:0 0 0 0;
}
</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>qrc/search.png</normaloff>qrc/search.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QListWidget" name="music_playlist">
      <property name="styleSheet">
       <string notr="true">border:none;
border-top:1px solid rgb(179, 179, 179);
border-bottom:1px solid rgb(179, 179, 179);</string>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QFrame" name="frame_2">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>70</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_7">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="1" column="1">
        <widget class="QPushButton" name="play_music">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">.QPushButton {
  background-color: rgb(255, 255, 255, 0);
  border: none;
  color: #1A6DFF;
  text-align: center;
  font-size: 22px;
  padding: 5px;
  margin:0 0 0 0;
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>qrc/play.png</normaloff>qrc/play.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="11">
        <widget class="QSlider" name="music_sound_slider">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="sliderPosition">
          <number>50</number>
         </property>
         <property name="tracking">
          <bool>true</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QLabel" name="current_music_file_name">
         <property name="styleSheet">
          <string notr="true">  background-color: rgb(255, 255, 255, 0);
  border: none;
  color: #1A6DFF;
  text-align: left;
  font-size: 18px;
  padding: 5px;
  margin:0 0 0 0;

</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="10">
        <widget class="QPushButton" name="music_sound">
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">.QPushButton {
  background-color: rgb(255, 255, 255, 0);
  border: none;
  color: #1A6DFF;
  text-align: center;
  font-size: 22px;
  padding: 5px;
  margin:0 0 0 0;
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>qrc/sound.png</normaloff>qrc/sound.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="9">
        <widget class="QLabel" name="music_point_2">
         <property name="text">
          <string>00:00:00</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="next_music">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">.QPushButton {
  background-color: rgb(255, 255, 255, 0);
  border: none;
  color: #1A6DFF;
  text-align: center;
  font-size: 22px;
  padding: 5px;
  margin:0 0 0 0;
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>qrc/next.png</normaloff>qrc/next.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QPushButton" name="add_music_file">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">.QPushButton {
  background-color: rgb(255, 255, 255, 0);
  border: none;
  color: #1A6DFF;
  text-align: center;
  font-size: 22px;
  padding: 5px;
  margin:0 0 0 0;
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>qrc/add_file.png</normaloff>qrc/add_file.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="prev_music">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">.QPushButton {
  background-color: rgb(255, 255, 255, 0);
  border: none;
  color: #1A6DFF;
  text-align: center;
  font-size: 22px;
  padding: 5px;
  margin:0 0 0 0;
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>qrc/prev.png</normaloff>qrc/prev.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="stop_music">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">.QPushButton {
  background-color: rgb(255, 255, 255, 0);
  border: none;
  color: #1A6DFF;
  text-align: center;
  font-size: 22px;
  padding: 5px;
  margin:0 0 0 0;
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>qrc/stop.png</normaloff>qrc/stop.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="7">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" colspan="12">
        <widget class="QSlider" name="music_slider">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="6">
        <widget class="QLabel" name="music_point">
         <property name="text">
          <string>00:00:00</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>ism</tabstop>
  <tabstop>play_video_4</tabstop>
  <tabstop>stop_music_2</tabstop>
  <tabstop>music_playlist</tabstop>
  <tabstop>next_music</tabstop>
  <tabstop>add_music_file</tabstop>
  <tabstop>music_sound</tabstop>
  <tabstop>play_music</tabstop>
  <tabstop>prev_music</tabstop>
  <tabstop>music_sound_slider</tabstop>
  <tabstop>music_slider</tabstop>
  <tabstop>stop_music</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
